pages/common/initdb.md

```diff
@@ -0,0 +1,8 @@
+# initdb
+
+> Create a PostgreSQL database on disk.
+> More information: <https://www.postgresql.org/docs/9.5/app-initdb.html>.
+
+- Create a database at /usr/local/var/postgres:
```

Comment:
Perhaps we can change it to - "Create a database at a given location." ? I think that is simple enough.